domains/anshulyadav.is-not-a.dev.json

```diff
@@ -0,0 +1,54 @@
+{
+  "description": "Full DNS config for anshulyadav.is-not-a.dev (excluding NS records)",
+  "domain": "is-not-a.dev",
+  "subdomain": "anshulyadav",
+
+  "owner": {
+    "repo": "https://github.com/anshulyadav32/anshulyadav-github",
+    "email": "anshuly89@gmail.com"
+  },
+
+  "record": {
+    "A": [
+      "20.193.141.86"
+    ],
+    "MX": [
+      {
+        "priority": 10,
+        "value": "mail.anshulyadav.is-not-a.dev"
+      }
+    ],
+    "TXT": [
+      "v=spf1 +a +mx +a:hardcore-cerf.20-193-141-86.plesk.page -all",
+      "v=DMARC1; p=quarantine; adkim=s; aspf=s"
+    ],
+    "CNAME": {
+      "www": "anshulyadav.is-not-a.dev",
+      "ftp": "anshulyadav.is-not-a.dev"
+    },
+    "SRV": {
+      "_imaps._tcp": "anshulyadav.is-not-a.dev",
+      "_pop3s._tcp": "anshulyadav.is-not-a.dev",
+      "_smtps._tcp": "anshulyadav.is-not-a.dev"
+    }
+  },
+
+  "alias": {
+    "mail": {
+      "A": ["20.193.141.86"]
+    },
+    "webmail": {
+      "A": ["20.193.141.86"]
+    },
+    "ipv4": {
```

Comment:
Invalid records, several records cannot be used in conjunction with other records and vice versa.